### PR TITLE
add characteristics list

### PR DESCRIPTION
**Pull request by stanislav-ostr**
example

### CODE REVIEW

**Comment by stanislav-ostr** [comment]
**Also I had to fix `frontend/src/store/queries/cars 25:8` because slint didn't allow commit**

File: frontend/src/common/types/characteristics-list/characteristics-list.ts [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+export type CharacteristicsGroupProps = {
+  name: string;
+  options: CharacteristicsOption[];
+};
+
+export type CharacteristicsOption = {
+  value: string;
+  name?: string;
+  color?: string;
+};
```
Comment (oleksandr-danylchenko) [inline frontend/src/common/types/characteristics-list/characteristics-list.ts]: I have a few questions for this type:
1. What will each field contain if they all are defined?
2. Why do we have a separate field for the color? Isn't it just one of the options

Comment (stanislav-ostr) [reply frontend/src/common/types/characteristics-list/characteristics-list.ts]: example2

Comment (stanislav-ostr) [reply frontend/src/common/types/characteristics-list/characteristics-list.ts]: Would I should add one component with its own type (value, name, color) for the first list and a second component with the type (only value) for other lists?

Comment (oleksandr-danylchenko) [reply frontend/src/common/types/characteristics-list/characteristics-list.ts]: Okay, after the screenshot it started to make sense. Could you please attack a link to it in the comment for the `CharacteristicsOption`?

**Review by oleksandr-danylchenko — APPROVED** [review]

**Pull request merged**